### DISCUSSION TITLE
Design resource for Bootstrap v5

### DISCUSSION THREAD

**Discussion by mss6989** [twbs/bootstrap#32302] [Show & Tell]
My team recently created a design toolkit for designing with Bootstrap in Adobe XD. The kit is available for free and includes all colors, text styles, and components aligned to the latest Bootstrap v5 specification. 

You can download and use the kit for free here - https://www.adobe.com/products/xd/features/ui-kits.html#bootstrap

**Comment by ghost** [comment]
Thank you for the share! I just got the kit. 😁
